src/ChunkMap.cpp

```diff
@@ -51,24 +51,10 @@ cChunkMap::cChunkMap(cWorld * a_World) :
 
 
 
-cChunkMap::~cChunkMap()
-{
-	cCSLock Lock(m_CSLayers);
```

Comment:
The problem is that if the Lock is protecting us against anything, then we are already in UB land.